.github/workflows/build-test-publish.yml

```diff
@@ -21,7 +21,7 @@ jobs:
             c-compiler: "cl.exe"
             cxx-compiler: "cl.exe"
             cmake-build-type: "Release"
-          - os: macos-12
+          - os: macos-13
```

Comment:
Maybe the rest of this file should be updated to include things like:
https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/blob/ed0534498647851b0ef330d25a94b939e9fe3743/.github/workflows/build-test-cxx.yml#L80-L83

Reply:
See if PR #430 makes the trick then.